### DISCUSSION TITLE
HFSS variable assignment!

### DISCUSSION THREAD

**Discussion by Abhi-SimuTech** [ansys/pyaedt#3671] [Q&A] [answered]
Hi,
I am trying to implement the following, but getting the error as shown below:

![Capture_1](https://github.com/ansys/pyaedt/assets/145813888/4b67e873-51d6-48c5-b886-ee45d5fc193c)

![Capture_2](https://github.com/ansys/pyaedt/assets/145813888/e549bab6-64c3-4548-8e14-84ed8fecd8c2)

Any suggestions will be greatly appreciated.

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @Abhi-SimuTech ,

Please next time provide the code, instead of a screenshot, because it takes time to write the code.

About the issue, this is a python escape character problem, you need to add the explicit quotes with the backslash (\).

m1[key] = '[\"' + value + '\"]' 

Then the final code is:

```
import pyaedt

m1 = pyaedt.Hfss(specified_version="2023.2", new_desktop_session=False)

params = {"gapZ": 0, "prefix": "_a1_"}

m1["prefix"] = '["_a1_"]'

for key, value in params.items():
    if not isinstance(value, str):
        m1[key] = "{}{}".format(value, "mm")
    else:
        m1[key] = '[\"' + value + '\"]' 

m1.release_desktop(False, False)
```
